server/src/DBTablesAction.h

```diff
@@ -173,9 +173,15 @@ struct ActionLog {
 	int      endTime;
 	int      failureCode;
 	int      exitCode;
+	ServerIdType serverId;
+	EventIdType  eventId;
 	uint32_t nullFlags;
 };
 
+typedef std::list<ActionLog>          ActionLogList;
```

Comment:
In this case, both of notations causes same result and have nothing advantages.

But, using type alias can handle `template` there:

```cpp
template<class Value>
using dict = std::map<std::string, Value>;
```

And function pointer become to be more readable:

```cpp
typedef void (*fp)(int, char);
using fp = void (*)(int, char);
```